Library/OcStorageLib/OcStorageLib.c

```diff
@@ -317,6 +317,14 @@ OcStorageFree (
   IN OUT OC_STORAGE_CONTEXT            *Context
   )
 {
+  if (Context->DummyStorageHandle != NULL) {
+    gBS->UninstallProtocolInterface (
```

Comment:
Please null the storage handle in case anybody calls this twice.

Reply:
Changed. Squashed with previous to keep the whole change as a single commit.